models/std_med_history.js

```diff
@@ -0,0 +1,14 @@
+import { Schema } from 'mongoose';
+import { connector } from './databaseUtil';
+
+const studentMedicalSchema= new Schema({
```

Comment:
studentMedicalSchema will be just a object no need for `new Schema`

Reply:
Done